client/src/api/users.js

```diff
@@ -0,0 +1,36 @@
+import apiRequest from './lib/apiRequest'
+
+const usersApiClient = {
+  current (apiOptions) {
+    return apiRequest.get('/user', apiOptions)
+  },
+
+  // TODO: add apiOptions
+  login (email, password) {
+    const options = {
+      method: 'POST',
+      headers: {
+        'Accept': 'application/json',
+        'Content-Type': 'application/json'
+      },
+      body: JSON.stringify({ email, password })
+    }
+
+    return fetch('/api/users/login', options)
+
+    // const body = { email, password }
+    // return apiRequest.post('/users/login', body)
+  },
+
+  signup (apiOptions, user) {
+    const body = { user }
+    return apiRequest.post('/users', apiOptions, body)
+  },
+
+  update (apiOptions, user) {
+    const body = { user }
+    return apiRequest.put('/user', apiOptions, body)
```

Comment:
Instead of a separate `/user` versus `/users`, how about something like `/users?user=${user}`? or even better `/users/${identifier}` with the identifier using the email and options body including other user info. That way we use the same base `/users` for all user-based routes. And then later when a user updates his or her email.... POST on `/users/${identifier}/email`.